server/index.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config()
+const { MONGODB_URI, PORT } = process.env
+
+// connect to mongodb via mongoose
+const mongoose = require('mongoose')
+const options = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}
+mongoose.connect(MONGODB_URI, options, () => {
+  console.log('Connected to MongoDB.')
+})
+
+// start the express server
+const app = require('./src/app')
+app.start(PORT)
```

Comment:
Here are some things that need to get done:
- [ ]  Can't add todo list, it is failing to fetch
- [ ]  I see that you have auth route in there, but I see no way to sign in/sign up for auth

Reply:
I think you aren't seeing it because I didn't merge this pull request